### PR TITLE
Migration guide 2.0 - 2.1

### PR DESCRIPTION
**Pull request by RayRoestenburg**
### What changes were proposed in this pull request?

Added a Migration guide from 2.0.x to 2.1.x.

### CODE REVIEW

**Review by marcospereira — APPROVED** [review]
LGTM!

File: docs/docs-source/docs/modules/project-info/pages/migration-2_0-2_1.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+= Migration Guide 2.0.x to 2.1.x
+
+include::ROOT:partial$include.adoc[]
+
+The most notable new feature in version 2.1.x is the native Flink and Spark integration support. 
+
+Migrating from 2.0.x to 2.1.x, it is required to update the Cloudflow operator, update the `kubectl-cloudflow` plugin, update the `sbt-cloudflow` plugin and rebuild your application.
```
Comment (thothmoses) [inline docs/docs-source/docs/modules/project-info/pages/migration-2_0-2_1.adoc:7]:
```suggestion
When migrating from 2.0.x to 2.1.x, it is required to update the Cloudflow operator, update the `kubectl-cloudflow` plugin, update the `sbt-cloudflow` plugin and rebuild your application.
```

File: docs/docs-source/docs/modules/project-info/pages/migration-2_0-2_1.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+If you want to take advantage of the native integration feature, you will need to take the following steps:
```
Comment (thothmoses) [inline docs/docs-source/docs/modules/project-info/pages/migration-2_0-2_1.adoc:9]:
```suggestion
To take advantage of the native integration feature, you will need to take the following steps:
```

File: docs/docs-source/docs/modules/project-info/pages/migration-2_0-2_1.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Uninstall the Flink and Spark operators with the commands below:
+
+----
+helm uninstall --namespace cloudflow flink-operator
+helm uninstall --namespace cloudflow spark-operator
+----
+
+Note: From version 2.0.20 and onward, Docker images are built on top of the standard `adoptopenjdk/openjdk8` image. Runtime specific base images usage has been deprecated (e.g. `cloudflowFlinkBaseImage`), if you are still using those, and need to customize the image, simply add additional instructions to `extraDockerInstructions`. Please note that, from version 2.0.20 and up, Cloudflow ships with a new kubectl-cloudflow plugin. From the utilization perspective, nothing changes, apart from the need for an = sign in between the streamlet name and the number of replicas in the `scale` subcommand.
```
Comment (thothmoses) [inline docs/docs-source/docs/modules/project-info/pages/migration-2_0-2_1.adoc:28]:
```suggestion
Note: From version 2.0.20 and onward, Docker images will bebuilt on top of the standard `adoptopenjdk/openjdk8` image. Runtime specific base images usage has been deprecated (for example, `cloudflowFlinkBaseImage`). If you are still using this type of image, and need to customize the image, simply add additional instructions to `extraDockerInstructions`. Please note that from version 2.0.20 and up, Cloudflow ships with a new kubectl-cloudflow plugin. From the utilization perspective, nothing changes, apart from the need for an = sign in between the streamlet name and the number of replicas in the `scale` subcommand.
```

**Review by thothmoses — APPROVED** [review]
A couple of minor tweaks

**Comment by RayRoestenburg** [comment]
Thanks for the quick review @marcospereira @thothmoses !

**Comment by RayRoestenburg** [comment]
(Fixed @thothmoses tweaks after merge, should eventually get published)

**Pull request merged**